### DISCUSSION TITLE
Badges to allow emoticons to be used

### DISCUSSION THREAD

**Discussion by habitats-tech** [Kovah/LinkAce#557] [Ideas & Feature Requests]
Create a 3rd categorisation option, named Badges, to allow emoticons to be used.

**Comment by Kovah** [comment]
Hi, thank you for taking time to write all your input. This is a very interesting idea. Should these badges, or like reactions, be added primarily by other users or only by yourself?

**Comment by habitats-tech** [comment]
My idea was for me to do it. However if you think this might be beneficial to Guest users as well, your decision will be better than mine. It all depends how much effort you are willing to excerpt. As an ex developer I will understand whichever way you want to swing things.
